packages/blocks/src/lib/navigation-map/stores.ts

```diff
@@ -0,0 +1,27 @@
+import * as THREE from 'three';
+import { writable } from 'svelte/store';
+import { currentWritable } from '@threlte/core';
+import type { Waypoint, Obstacle } from '$lib';
+
+export const hovered = currentWritable<string | null>(null);
+
+export const waypoints = currentWritable<Waypoint[]>([]);
+export const obstacles = currentWritable<Obstacle[]>([]);
+
+/** The currently selected tab. */
+export const tab = writable<'Obstacles' | 'Waypoints'>('Waypoints');
+
+/** The visible set of tabs. */
+export const tabs = writable<string[]>(['Obstacles', 'Waypoints']);
+
+/** If we're looking at obstacles in a 2d top-down or 3d orbiting view */
+export const view = currentWritable<'2D' | '3D'>('2D');
+
+/** Whether or not we can create obstacles */
+export const write = currentWritable(false);
```

Comment:
Can this be a more descriptive variable than `write`? e.g. `canCreateObstacles`, from the comment

Reply:
yes, was not a good name - replaced!